tests/microbenchmarks.py

```diff
@@ -111,11 +125,14 @@ def foo():
 
         for i in range(100000):
             foo()
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_code(test_case):
     print("Running", "test_code")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils